app/ts/components/pages/InterceptorAccess.tsx

```diff
@@ -190,16 +190,16 @@ export function InterceptorAccess() {
 				const currentTabId = (await browser.tabs.getCurrent()).id
 				if (currentWindowId === undefined) throw new Error('could not get current window Id!')
 				if (currentTabId === undefined) throw new Error('could not get current tab Id!')
-				browser.windows.update(currentWindowId, { focused: true })
-				browser.tabs.update(currentTabId, { active: true })
 				return
 			}
 		}
 		browser.runtime.onMessage.addListener(popupMessageListener)
-		sendPopupMessageToBackgroundPage({ method: 'popup_interceptorAccessReadyAndListening' })
 		return () => browser.runtime.onMessage.removeListener(popupMessageListener)
 	})
 
+	
+	useEffect(() => { sendPopupMessageToBackgroundPage({ method: 'popup_interceptorAccessReadyAndListening' }) }, [])
```

Comment:
`[]` as second parameter to `useEffect` means it will run once when the component is first instantiated.  What does missing parameter mean (like the previous `useEffect`)?  The docs don't seem to specify.  Is it the same as `[]`?  If so, we should probably be consistent on which one we use.

Reply:
they are not the same, [] means it runs once, and missing parameter means its run everytime the component refreshes

Reply:
you can read about that eg in here: <https://stackoverflow.com/questions/55240526/useeffect-second-argument-variations-in-react-hook>